### PR TITLE
[14.0][MIG] hr_holidays_settings

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
@ForgeFlow
Migration to 14.0

This module adds a Settings form in the menu of Leaves (HR Holidays module).
This settings are empty by default but enable other modules to be able to extend it.

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM!!

File: hr_holidays_settings/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "HR Holidays Settings",
+    "summary": "Enables Settings Form for HR Holidays.",
+    "version": "14.0.1.1.0",
```
Comment (MiquelRForgeFlow) [inline hr_holidays_settings/__manifest__.py]:
```suggestion
    "version": "14.0.1.0.0",
```

File: hr_holidays_settings/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+* Miquel Raïch <miquel.raich@eficent.com>
```
Comment (MiquelRForgeFlow) [inline hr_holidays_settings/readme/CONTRIBUTORS.rst]:
```suggestion
* Miquel Raïch <miquel.raich@forgeflow.com>
```

File: hr_holidays_settings/__manifest__.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "HR Holidays Settings",
     "summary": "Enables Settings Form for HR Holidays.",
-    "version": "12.0.1.1.0",
+    "version": "14.0.1.1.0",
     "category": "Human Resources",
     "website": "https://github.com/OCA/hr",
     "author": "Eficent, Odoo Community Association (OCA)",
```
Comment (JoanMForgeFlow) [inline hr_holidays_settings/__manifest__.py]: ForgeFlow

**Review by JoanMForgeFlow — APPROVED** [review]
Minor change, LGTM!

File: hr_holidays_settings/__manifest__.py (hunk 1 above)
Comment (JuanyDForgeflow) [reply hr_holidays_settings/__manifest__.py]: fixed

File: hr_holidays_settings/readme/CONTRIBUTORS.rst (hunk 2 above)
Comment (JuanyDForgeflow) [reply hr_holidays_settings/readme/CONTRIBUTORS.rst]: fixed

File: hr_holidays_settings/__manifest__.py (hunk 3 above)
Comment (JuanyDForgeflow) [reply hr_holidays_settings/__manifest__.py]: fixed

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by albariera — APPROVED** [review]
LGMT 👍

**Pull request merged**